_component_design/toast.md

```diff
@@ -48,15 +44,19 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/toasts/
 
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-8-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best Practices
 
-### Best Practices
+Consider the following best practices when you construct toast:
 
-- If there's an error and the user can take an action to correct it, use a call to action (CTA).
-- Use complete sentences. If only one sentence is used, do not include a period. However, if there are two or more sentences, use periods.
-- Try not to exceed more than four lines of your message (excluding the CTA).
-- Only use one CTA per toast. Never include “Dismiss” or “Close” as a CTA.
-- Clicking on the “X” closes the toast notification. Hovering over it keeps it open until the cursor moves off of it. (See [Timing]({{page.url}}#timing) section for more detail.)
+- A toast can include an optional call-to-action (CTA) that navigates the user out of the current page, or that triggers an action.
```

Comment:
Maybe focus on how this is an edge case.
While most toasts do not need a CTA, you may want to include one. 
reasons to include a CTA:
- user has navigated away from the page where the process what initiated and needs to navigate easily back to continue with the job to be done.  E.g. user created a new user, if the process took some time, and they navigated away, they may want to navigate back to that user to configure their permission settings. in this case, a link that navigates the the new user's details page may be useful.
- if the process failed it's may be helpful to include a CTA to retry the process

Reply:
Done.